src/python/grpcio_tests/tests_aio/unit/interceptor_test.py

```diff
@@ -685,6 +685,110 @@ class Interceptor(aio.UnaryUnaryClientInterceptor):
                 self.fail("Callback was not called")
 
 
+class _LoggingServerInterceptor(aio.ServerInterceptor):
+
+    def __init__(self, tag, record):
+        self.tag = tag
+        self.record = record
+
+    async def intercept_service(self, continuation, handler_call_details):
+        self.record.append(self.tag + ':intercept_service')
+        return await continuation(handler_call_details)
+
+
+class _GenericServerInterceptor(aio.ServerInterceptor):
+
+    def __init__(self, fn):
+        self._fn = fn
+
+    async def intercept_service(self, continuation, handler_call_details):
+        return await self._fn(continuation, handler_call_details)
+
+
+def _filter_server_interceptor(condition, interceptor):
+    async def intercept_service(continuation, handler_call_details):
+        if condition(handler_call_details):
+            return await interceptor.intercept_service(continuation,
+                                                       handler_call_details)
+        return await continuation(handler_call_details)
+
+    return _GenericServerInterceptor(intercept_service)
+
+
+class TestServerInterceptor(AioTestBase):
```

Comment:
I would like to have a test for checking observability of the code responses sent by the handler, in the same way as we have for the channel [1]

[1] https://github.com/grpc/grpc/blob/master/src/python/grpcio_tests/tests_aio/unit/interceptor_test.py#L93

Reply:
Server interceptors are wrapped at `find_method_handler` , the interceptors won't get the status code of response.